### PR TITLE
Day1

### PR DESCRIPTION
**Pull request by Ismailco**
- Create the mobile version of the festival page
- Used CSS, Flexbox to design layout
- Used semantic HTML
- Create Desktop Version
- Used JavaScript to add participants dynamically
- This is a [link to the video that the project](https://www.loom.com/share/e40533729962471ba4047609d60b4a28)

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+.headline_place {
+  font-family: lato, sans-serif;
+  font-size: 15px;
+  color: #272a31;
+}
+
+/* program section */
+
+.main_section {
+  background-image: url(../img/Photo\ by\ Andrew\ Ridley\ on\ Unsplash.png);
```
Comment (brytebee) [inline styles/style.css]: - [x] **Required:** You have done an awesome job. However, the background image here isn't similar to the original design.

Your Design | Behance Design
:---: | :---:
image | image


**Hint:**  Use this [resource](https://cssgradient.io/) to easily generate a similar one.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+          <img class="partner_logo" src="img/partners/partner2.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner3.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner4.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner5.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner6.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner7.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner8.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner9.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner10.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner11.png" alt="partner_logo" />
```
Comment (brytebee) [inline index.html:116]: - [x] **Required:** Consider the color scheme of this section. Some of the icons used here are not visible. Kindly comply with the color scheme used in the Template Design

Your Design | Behance Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Ismailco 👋, 

### Highlights
- awesome UI design 👏
- good use of JS objects and arrays 👏
- no linters/no console errors 👏
- well written README documentation 👏

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ReshyResh — APPROVED** [review]
# Hi @Ismailco , 

### Good job addressing the changes requested by the previous reviewer.
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**